crates/client/src/admin_websocket.rs

```diff
@@ -296,6 +312,16 @@ impl AdminWebsocket {
         }
     }
 
+    pub async fn register_dna(&self, payload: RegisterDnaPayload) -> ConductorApiResult<DnaHash> {
```

Comment:
This call can be deprecated, I think. Not on this PR, just as a thought.

Reply:
It was intended for HoloPorts, I think, so that you can upload a DNA and then instantiate agent cells without having to upload the DNA over and over again.

Reply:
Agreed, I've never seen a reason to use this.

I think it's fine to add these extra admin calls while they're available on the conductor but they were left out because nobody uses them. We left out some things that people were using and those have been added to the Rust client over time.